testing/tests/DevExpress.exporter/jspdfParts/jspdf.dataGrid.tests.js

```diff
@@ -30,51 +29,78 @@ const moduleConfig = {
     }
 };
 
-QUnit.module('Scenarios, generate autoTable options', moduleConfig, () => {
-    const getOptions = (context, dataGrid, expectedCustomizeCellArgs, options) => {
-        const { /* keepColumnWidths = true, selectedRowsOnly = false, */ autoTableOptions = {} } = options || {};
+QUnit.module('Scenarios, check autoTableOptions', moduleConfig, () => {
+    const getOptions = (context, dataGrid, options) => {
+        const { keepColumnWidths = true, /* selectedRowsOnly = false, */ autoTableOptions = {} } = options || {};
 
         const result = {
             component: dataGrid,
             jsPDFDocument: context.jsPDFDocument
         };
+        result.keepColumnWidths = keepColumnWidths;
         result.autoTableOptions = autoTableOptions;
         return result;
     };
 
     QUnit.test('Empty grid', function(assert) {
         const done = assert.async();
         const dataGrid = $('#dataGrid').dxDataGrid({}).dxDataGrid('instance');
-        const expectedCells = [];
 
-        exportDataGrid(getOptions(this, dataGrid, expectedCells)).then((jsPDFDocument) => {
+        exportDataGrid(getOptions(this, dataGrid)).then((jsPDFDocument) => {
             const autoTableOptions = jsPDFDocument.autoTable.__autoTableOptions;
-            helper.checkTableAndColumnWidths(undefined, [], autoTableOptions);
             helper.checkCellsContent([], [], autoTableOptions);
             done();
         });
     });
 
     QUnit.test('Simple grid', function(assert) {
         const done = assert.async();
-        const ds = [{ id: 1, name: 'test' }];
+        const ds = [{ prop1: 'text1' }];
         const dataGrid = $('#dataGrid').dxDataGrid({
             dataSource: ds,
-            keyExpr: 'id',
-            columns: [
-                { dataField: 'id', width: 50, caption: 'id' },
-                { dataField: 'name', caption: 'name' }
-            ],
+            columns: [{ dataField: 'prop1', caption: 'f1' }],
             loadingTimeout: undefined,
-            showColumnHeaders: true,
-            width: 500
+            showColumnHeaders: true
         }).dxDataGrid('instance');
-        const expectedCells = [];
-        exportDataGrid(getOptions(this, dataGrid, expectedCells)).then((jsPDFDocument) => {
+
+        exportDataGrid(getOptions(this, dataGrid)).then((jsPDFDocument) => {
             const autoTableOptions = jsPDFDocument.autoTable.__autoTableOptions;
-            helper.checkTableAndColumnWidths(Export.convertPixelsToPoints(500), [Export.convertPixelsToPoints(50), 'auto'], autoTableOptions);
-            helper.checkCellsContent([['id', 'name']], [['1', 'test']], autoTableOptions);
+            helper.checkCellsContent([['f1']], [['text1']], autoTableOptions);
```

Comment:
What about scenarios when will be more cells content. Does we may to keep the expectedCells variable for this? I suggest we will need it in the near future. It is cool when the structure of code blocks is consistent for each test.

Reply:
We discussed it by Vova. We are going to return to it later.